src/decorators.ts

```diff
@@ -53,8 +62,28 @@ export function Column(options: ColumnOption | DataTypes<DataType> = {}) {
     // target refers to model prototype, an internal instance of `Bone {}`
     const model = target.constructor;
     const { attributes = (model.attributes = {}) } = model;
-    const { name: columnName, ...restOptions } = options;
+    const { name: columnName, setter, getter, ...restOptions } = options;
     attributes[propertyKey] = { ...restOptions, columnName };
+
+    const customDescriptor: PropertyDescriptor = {};
+    if(setter && typeof setter === 'function') {
+      customDescriptor.set = setter;
+    }
+
+    if(getter && typeof getter === 'function') {
+      customDescriptor.get = getter;
+    }
+
+    if(!Object.keys(customDescriptor).length) return;
+    
+    customDescriptor.enumerable = true;
+    customDescriptor.configurable = true;
+
+    const defaultDescriptor = Object.getOwnPropertyDescriptor(target, propertyKey);
+    Object.defineProperty(target, propertyKey, {
+      ...defaultDescriptor,
+      ...customDescriptor,
+    })
```

Comment:
are we going to support custom `getter`/`setter`s with `{ getter() {}, setter() {} }`? I think we've already got `{ get() {}, set() {} }` covered.